.github/workflows/check.yml

```diff
@@ -10,7 +10,7 @@ concurrency:
 
 jobs:
   check:
-    name: ${{ vars.CI_IMAGE }}
+    name: Check 📦
```

Comment:
no major gripe here: just that when this will run it will be show Checks/Check/Check instead of Checks/check/image_name; which might be slightly more informative